.github/workflows/check_changelog.yml

```diff
@@ -0,0 +1,15 @@
+name: Check release notes
+
+on:
+  pull_request:
+    branches:
+      - 'main'
+
+jobs:
+  Check-Changelog:
+    name: Check Changelog Action
+    runs-on: ubuntu-20.04
```

Comment:
I got this email from Github yesterday: 

![image](https://github.com/user-attachments/assets/d82dca18-68c5-40db-bc2a-b8ce8ee537cf)

Better to use `ubuntu-latest`, probably?

Reply:
Yes, I will probably have to update some extra things because Ubuntu 24.04 does not support Python 3.7 and it is the minimum version for the library. I think 3.8 is still available on Ubuntu 24.04.

Reply:
3.7 and 3.8 are both end of life: https://devguide.python.org/versions/#unsupported-versions